src/main/java/de/retest/recheck/util/OptionalUtil.java

```diff
@@ -0,0 +1,22 @@
+package de.retest.recheck.util;
+
+import java.util.Optional;
+import java.util.function.Supplier;
+import java.util.stream.Stream;
+
+public class OptionalUtil {
```

Comment:
If we only could use [`@ExtensionMethod`](https://projectlombok.org/features/experimental/ExtensionMethod)

Reply:
Or [Kotlin](https://kotlinlang.org/docs/reference/extensions.html). 😉 But yes, I vote against experimental Lombok features in prod.